build/BundledRuntimes.props

```diff
@@ -34,9 +34,10 @@
 
   <PropertyGroup>
     <AspNetCoreRuntimeInstallerBlobRootUrl>$(CoreSetupBlobRootUrl)aspnetcore/store/$(AspNetCoreRuntimeAzureblobStoresSubfolderName)</AspNetCoreRuntimeInstallerBlobRootUrl>
+    <AspNetCoreRuntimeBuildDropFeed>$(AspNetCoreRuntimeInstallerBlobRootUrl)/packages/index.json</AspNetCoreRuntimeBuildDropFeed>
```

Comment:
Why do we need to keep adding feeds for ASP.NET? Who will remove old ones when they are not needed anymore? Will the old ones be deprecated at some point?

Reply:
We're trying to get off myget, and yes, hopefully the old ones will be completely unnecessary in the future. I can try removing them in this PR to see if anything else is still being pulled from aspnet myget feeds.